### PR TITLE
add parameterized query to gRPC Java example

### PR DESCRIPTION
**Pull request by jeffreyscarpenter**
Add an example of creating a parameterized query to the gRPC examples

**Checklist**
- [x] Changes manually tested
- [ ] Automated Tests added/updated
- [x] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Review by versaurabh — APPROVED** [review]

**Review by polandll — APPROVED** [review]

File: grpc-examples/src/main/java/io/stargate/grpcexample/GrpcClientExecuteQuery.java [hunk 1]
```diff
@@ -221,6 +223,35 @@ public void executeSingleQuery() {
...
+    response =
+        blockingStub.executeQuery(
+            QueryOuterClass.Query.newBuilder()
+                .setCql("SELECT k, v FROM ks.test WHERE k = ?")
+                .setValues(QueryOuterClass.Values.newBuilder().addValues(Values.of("b")).build())
+                .build());
+
+    QueryOuterClass.ResultSet rs = response.getResultSet();
+
+    System.out.println(
```
Comment (tatu-at-datastax) [inline grpc-examples/src/main/java/io/stargate/grpcexample/GrpcClientExecuteQuery.java:249]: I assume these were accidentally left during testing? Should be removed.

**Review by tatu-at-datastax — CHANGES_REQUESTED** [review]

**Comment by tatu-at-datastax** [comment]
Makes sense, but I think those debug print statements should be removed.

File: grpc-examples/src/main/java/io/stargate/grpcexample/GrpcClientExecuteQuery.java (hunk 1 above)
Comment (jeffreyscarpenter) [reply grpc-examples/src/main/java/io/stargate/grpcexample/GrpcClientExecuteQuery.java:249]: Remember that this is example code that the user is meant to run (likely in their IDE) after setting the Astra token.  It's supposed to print stuff out.

**Review by tatu-at-datastax — APPROVED** [review]

File: grpc-examples/src/main/java/io/stargate/grpcexample/GrpcClientExecuteQuery.java (hunk 1 above)
Comment (tatu-at-datastax) [reply grpc-examples/src/main/java/io/stargate/grpcexample/GrpcClientExecuteQuery.java:249]: Ah. Ok makes sense then. Forgot the context. Thanks!

**Review by tatu-at-datastax — APPROVED** [review]

**Pull request merged**